fe/fe-core/src/main/java/com/starrocks/sql/ast/DataCacheSelectStatement.java

```diff
@@ -29,6 +30,7 @@ public class DataCacheSelectStatement extends DdlStmt {
 
     // Set after DataCacheAnalyzer analyze properties
     private boolean isVerbose = false;
+    private String catalog = InternalCatalog.DEFAULT_INTERNAL_CATALOG_NAME;
```

Comment:
add some comment to explain why need this variable

Reply:
done